dcpquery/_config.py

```diff
@@ -66,7 +66,10 @@ def db(self):
                 @sqlalchemy.event.listens_for(self._db, 'before_cursor_execute', retval=True)
                 def before_cursor_execute(conn, cursor, statement, parameters, context, executemany):
                     if statement.startswith("INSERT"):
-                        statement += " ON CONFLICT DO NOTHING"
+                        if "RETURNING" in statement:
+                            statement.replace("RETURNING", "ON CONFLICT DO NOTHING RETURNING")
+                        else:
+                            statement += " ON CONFLICT DO NOTHING"
```

Comment:
This is topically separate from the rest of the PR. Consider splitting it into a separate PR stacked underneath this one.

Reply:
👍